src/libraries/System.Private.CoreLib/src/System/Globalization/StringInfo.cs

```diff
@@ -131,7 +134,8 @@ public static string GetNextTextElement(string str, int index)
 
         /// <summary>
         /// Returns the length of the first text element (extended grapheme cluster) that occurs in the input string
-        /// starting at the specified index.
+        /// starting at the specified index. A grapheme cluster is a sequence of one or more Unicode
```

Comment:
Should the grapheme cluster explanation be under a `remarks` section?

Reply:
OK sure.